jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/EditFunctionDialog.kt

```diff
@@ -155,7 +155,7 @@ class EditFunctionDialog(
 
         val regionProvider = AwsRegionProvider.getInstance()
         val settings = ProjectAccountSettingsManager.getInstance(project)
-        view.setXrayControlVisibility(regionProvider.isServiceSupported(settings.activeRegion, "xray"))
+        view.setXrayControlVisibility(mode !== UPDATE_CODE && regionProvider.isServiceSupported(settings.activeRegion, "xray"))
```

Comment:
Other checks in this method use `==` on `mode`. Are you sure you don't want to use `!=` instead of `!==` ?

Reply:
Fixed this. Not sure the reason why we used `==` for an enum, but to be consistent, I updated.